ui.frontend/src/view/FormFieldBase.js

```diff
@@ -352,6 +352,10 @@ export default class FormFieldBase extends FormField {
             }
         }
     }
+    
+    updateActiveChild(activeChild) {
```

Comment:
Please write a test case for this use-case

Reply:
done